docs/source/developer_guide/guides/3_simple_cpp_stage.md

```diff
@@ -336,21 +336,20 @@ PYBIND11_MODULE(morpheus_example, m)
 
 ### Python Changes
 
-We need to make a few minor adjustments to our Python implementation of the `PassThruStage`. First, we import the new `morpheus_example` Python module we created in the previous section.
-
-```python
-from _lib import morpheus_example as morpheus_example_cpp
-```
+We need to make a few minor adjustments to our Python implementation of the `PassThruStage`.
 
 As mentioned in the previous section, we will need to change the return value of the `supports_cpp_node` method to indicate that our stage now supports a C++ implementation.  Our `_build_single` method needs to be updated to build a C++ node when `morpheus.config.CppConfig.get_should_use_cpp()` is `True` using the `self._build_cpp_node()` method. The `_build_cpp_node()` method compares both `morpheus.config.CppConfig.get_should_use_cpp()` and `supports_cpp_node()` and returns `True` only when both methods return `True`.
 
+ We lazily import the new `morpheus_example` Python module we created in the previous section only when we need to build a C++ node since importing a compiled extension module comes at a performance cost.
```

Comment:
Uhm... not sure I agree as "... importing a compiled extension" makes it clear it is the importing we are concerned with, and the previous parts of the document discusses the performance advantages of creating the C++ extension.

Reply:
> importing a compiled extension module comes at a performance cost
Two people who read this might come up with a different mental model for how imports effect the performance of Morpheus. It's possible that importing a module produces a side effect, and it's ambiguous as to whether the import of this module causes a side effect that has a permanent effect on performance. I think it's important that the reader understand that the performance cost is limited to the import statement itself and does not have a lasting effect on pipeline execution.